README.md

```diff
@@ -63,6 +64,8 @@ named `example.spec.js`.
 
 This has been tested using `node v10.11.0` and `npm 6.4.1`.
```

Comment:
@pranshukharkwal I think you can mention this version as `minimum version required for node`

Reply:
Alright. I will do this in the next commit. Thanks